### PR TITLE
Update underlying SDK versions

### LINKED ISSUE #798: Any interest in an update to FBSDK 8.0?
Hello!

I have the underlying native SDKs updated to 8.0 in my work project for testing (it's early in my release cycle) and everything seems fine.

Interested in a PR to implement that here? It's just a bump in build.gradle and the podspec it seems, so nothing big but it would be great to see if it works for others or if I'm missing something

### PR DESCRIPTION
**Pull request by mikehardy**
Allow facebook-android-sdk to be 7.0 through 8.x (should get 8.2.0 now)
Use facebook-ios-sdk 8.1 (current stable)

Fixes #798

Test Plan:

I updated the underlying SDKs, and I have integrated changes that do the same (via patch-package) into my work app. Everything *seems* to work, but I do not use all of the SDK functionality.

I would assume (*dangerous*) if it compiles okay that it should be fine, if the APIs and symbols defined upstream did not change meaning

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -45,7 +45,7 @@ repositories {
     jcenter()
 }
 
-def FACEBOOK_SDK_VERSION = safeExtGet('facebookSdkVersion', '[7.0,8.0[')
+def FACEBOOK_SDK_VERSION = safeExtGet('facebookSdkVersion', '[7.0,9.0[')
```
Comment (yum650350) [inline android/build.gradle:48]: Should it be `'[7.0,9.0)'` ?

According to https://developers.facebook.com/docs/app-events/upgrade-guide?locale=en_US

Comment (mikehardy) [reply android/build.gradle:48]: You are right - I'm not that great at the gradle version ranges, and I think we want all of 8.x but *exclusive* of 9 which is ')' thank you - I will update and re-push

https://docs.gradle.org/current/userguide/single_versions.html

**Comment by mikehardy** [comment]
@janicduplessis this is probably good for review now, you appear to be the merger in this module? Cheers :-)

**Comment by janicduplessis** [comment]
Hey thanks @mikehardy , I think these changes are already in https://github.com/facebook/react-native-fbsdk/pull/804 which we should be able to merge soon.

**Comment by mikehardy** [comment]
@janicduplessis I think you are right but I noted on that PR that there are two things worth changing perhaps https://github.com/facebook/react-native-fbsdk/pull/804/files?file-filters%5B%5D=.gradle&file-filters%5B%5D=.h&file-filters%5B%5D=.java&file-filters%5B%5D=.m&file-filters%5B%5D=.podspec#r521603372

I'll close this in preference to that one :-)

**Comment by janicduplessis** [comment]
Thanks, perfect!

**Pull request closed without merging**